src/Debug.h

```diff
@@ -214,10 +217,12 @@ class Raw
 
 private:
     void printHex(std::ostream &os) const;
+    size_t printableSize() const { return (limit_ > 0 && limit_ < size_ ? limit_ : size_); }
```

Comment:
AFAICT, after the other changes, this method should be removed while limit_ is renamed to printableSize_. It is cheaper and clearer to compute this value correctly once than to call this method two-three times in a row. This change will also clarify what `limit_` is to the code reader (without the need to consult the member declaration).

BTW, the atMost() method name should stay the same. From the caller point of view, that method sets a limit which may (and often will!) exceed `size_`. The _implementation_ of that method should set `printableSize_` correctly, of course.

Reply:
Done in f06e16fe5